src/test/suite/fixtures/recorded/surroundingPair/takeFirstChar.yml

```diff
@@ -0,0 +1,24 @@
+spokenForm: take first char
```

Comment:
I think this file is in the wrong directory, no?

Reply:
Correct. Eye tracker error
I'm going to add the directory name to the message